resources/views/themes/default1/user/2faModals.blade.php

```diff
@@ -43,7 +43,8 @@
 
           </div>
             <div class="col-md-2">
-                    <span style="font-size: 20px; pointer-events: initial; cursor: pointer; display: block;" id="copyBtn" title="Click to copy to clipboard" onclick="copyRecoveryCode()"><i class="fa fa-clipboard"></i></span>
+              <span class="badge badge-success badge-xs pull-right" id="copied" style="display:none;margin-top:-15px;margin-left:-20px;position: absolute;">Copied</span>
+                    <span style="pointer-events: initial; cursor: pointer; display: block;" id="copyBtn" title="Click to copy to clipboard" onclick="copyRecoveryCode()"><i class="fa fa-clipboard"></i></span>
```

Comment:
Please make it a useable component and use it everywhere

Reply:
Fixed